ui/src/app/views/application/show/home/application.home.component.ts

```diff
@@ -13,25 +13,24 @@ import { Project } from '../../../../model/project.model';
     templateUrl: './application.home.html',
     styleUrls: ['./application.home.scss']
 })
-export class ApplicationHomeComponent implements OnInit {
-
+export class ApplicationHomeComponent implements OnInit, OnChanges {
     @Input() project: Project;
     @Input() application: Application;
 
     dashboards: Array<GraphConfiguration>;
-    ready = false;
     overview: Overview;
 
     constructor(
         private _translate: TranslateService,
         private store: Store
-    ) {
-
-    }
+    ) { }
 
     ngOnInit(): void {
         this.store.dispatch(new FetchApplicationOverview({ projectKey: this.project.key, applicationName: this.application.name }));
         this.dashboards = new Array<GraphConfiguration>();
+    }
+
+    ngOnChanges(): void {
         this.overview = this.application.overview;
```

Comment:
Why no just use this.application.overview, and delete `this.overview` and push dashboard creation in the setter of application. I think it would be better than use ngOnChanges